src/integrationTest/java/uk/gov/hmcts/cmc/claimstore/controllers/ResendStaffNotificationsCoreCaseDataTest.java

```diff
@@ -0,0 +1,248 @@
+
+    @MockBean
+    protected SendLetterApi sendLetterApi;
+
+    @Captor
+    private ArgumentCaptor<EmailData> emailDataArgument;
+
+    private static final String PAGE = "1";
+
+    @Before
+    public void setUp() {
+        given(pdfServiceClient.generateFromHtml(any(byte[].class), anyMap()))
+            .willReturn(new byte[]{1, 2, 3, 4});
+        UserDetails userDetails = SampleUserDetails.getDefault();
+        User user = new User(AUTHORISATION_TOKEN, userDetails);
+        given(userService.getUserDetails(AUTHORISATION_TOKEN)).willReturn(userDetails);
+        given(userService.authenticateAnonymousCaseWorker()).willReturn(user);
+        given(userService.getUser(AUTHORISATION_TOKEN)).willReturn(user);
+        given(authTokenGenerator.generate()).willReturn(SERVICE_TOKEN);
+    }
+
+    @Test
+    public void shouldRespond404WhenClaimDoesNotExist() throws Exception {
+        final String nonExistingClaimReference = "something";
+        final String event = "claim-issue";
+        given(coreCaseDataApi.searchForCitizen(
+            any(),
+            any(),
+            any(),
+            any(),
+            any(),
+            eq(ImmutableMap.of("case.referenceNumber", CASE_REFERENCE, "page", PAGE, "sortDirection", "desc"))
+            )
+        ).willReturn(listOfCaseDetails());
+        makeRequest(nonExistingClaimReference, event).andExpect(status().isNotFound());
+        verify(emailService, never()).sendEmail(any(), any());
+    }
+
+    @Test
+    public void shouldRespond404WhenEventIsNotSupported() throws Exception {
+        final String nonExistingEvent = "some-event";
+
+        given(coreCaseDataApi.searchForCitizen(
+            any(),
+            any(),
+            any(),
+            any(),
+            any(),
+            eq(ImmutableMap.of("case.referenceNumber", CASE_REFERENCE, "page", PAGE, "sortDirection", "desc"))
+            )
+        ).willReturn(listOfCaseDetails());
+        makeRequest(CASE_REFERENCE, nonExistingEvent).andExpect(status().isNotFound());
+        verify(emailService, never()).sendEmail(any(), any());
+    }
+
+    @Test
+    public void shouldRespond409AndNotProceedForClaimIssuedEventWhenClaimIsLinkedToDefendant() throws Exception {
+        final String event = "claim-issued";
+
+        given(coreCaseDataApi.searchForCitizen(
+            any(),
+            any(),
+            any(),
+            any(),
+            any(),
+            eq(ImmutableMap.of("case.referenceNumber", CASE_REFERENCE, "page", PAGE, "sortDirection", "desc"))
+            )
+        ).willReturn(listOfCaseDetailsWithDefendant());
+
+        makeRequest(CASE_REFERENCE, event).andExpect(status().isConflict());
+
+        verify(emailService, never()).sendEmail(any(), any());
+    }
+
+    @Test
+    public void shouldRespond200AndSendNotificationsForClaimIssuedEvent() throws Exception {
+        final String event = "claim-issued";
+        final Claim claim = SampleClaim.builder()
```

Comment:
Please remove as this is never used as a claim, only `CASE_REFERENCE` is in use.